api/src/middlewares/verifyGoogleToken.js

```diff
@@ -0,0 +1,52 @@
+import HttpStatus from 'http-status-codes';
+import * as googleAuth from 'google-auth-library';
+
+import { GSUITE_DOMAIN } from '../constant';
+
+const OAuth2Client = new googleAuth.OAuth2Client(process.env.GOOGLE_CLIENT_ID, process.env.GOOGLE_SECRET);
+
+/**
+ * Validate the users' google id using google-auth-library.
+ *
+ * @param {Object} req
+ * @param {Object} res
+ * @param {function} next
+ *
+ */
+const validateGoogleToken = async (req, res, next) => {
+  try {
+    const ticket = await OAuth2Client.verifyIdToken({ idToken: req.body.tokenId });
+    const payload = ticket.getPayload();
+
+    let data;
```

Comment:
I think we can remove this from here and initialize it where it is being used

Reply:
done